test_client/src/main/java/io/reactivesocket/tck/TestSubscriber.java

```diff
@@ -201,6 +202,7 @@ public void onNext(T t) {
         Payload p = (Payload) t;
```

Comment:
I copied it from the RXjava test subscriber (and modified a few small things), but it doesn't really *do* anything by itself. It just handles the commands that the user gives, so as long as the user follows the RS specs, and by extension the parser, I think it should be ok.